ui/app/adapters/auth-method.js

```diff
@@ -55,7 +54,7 @@ export default ApplicationAdapter.extend({
       // ember data doesn't like 204s if it's not a DELETE
       data.config.id = path; // config relationship needs an id so use path for now
       return {
-        data: assign({}, data, { path: path + '/', id: path }),
+        data: { ...data, ...{ path: path + '/', id: path } },
```

Comment:
These can be just added to the spread object instead of spread on an object themselves, eg:
```suggestion
        data: { ...data, path: path + '/', id: path },
```